src/Common/TranslateRepositoryUrlGitTask.cs

```diff
@@ -66,7 +67,16 @@ static bool isMatchingHostUri(Uri hostUri, Uri uri)
                 return url;
             }
 
-            TranslatedRepositoryUrl = translate(RepositoryUrl);
+            try
+            {
+                TranslatedRepositoryUrl = translate(RepositoryUrl);
+            }
+            catch (NotSupportedException e)
+            {
+                Log.LogError(e.Message);
+                return;
+            }
```

Comment:
Is it ok to just log and return? I'm thinking that the error might be observer only a few week/months later when once debugs the code and sees that source linking doesn't work.
Why not let the exception propagate?

Reply:
I am fairly hazy on the specifics now, but I think logging the error results in an MsBuild error that can't be suppressed. Is the concern that since it is a post build action that error may not be noticed since the files will be on disk? I think the exception being thrown would have the same limitation. 

I don't remember the specific use case where the NotSupportedException was thrown to say whether or not the message is helpful enough to tell the user that their URL scheme/format is not supported by Sourcelink. But I thought those were the errors being thrown by the individual translators when they didn't support the repo url.